### DISCUSSION TITLE
How to solve the Buildroot Image building process in WSL2

### DISCUSSION THREAD

**Discussion by chu551122** [polarfire-soc/polarfire-soc-documentation#239] [Linux build systems for PolarFire SoC] [answered]
Dear Sir:
   I tried to build the buildroot image for polarfire soc icicle kit in WSL Ubuntu 20.04 but got the below error message. I didn't modify anything that I clone from the github and I can build successfully in Virtualbox Ubuntu 20.04.
  Besides, how to program the Buildroot image into SDCARD in WSL2?
  Thank you very much!
 
![image](https://user-images.githubusercontent.com/55179520/201266677-54b75360-2fd7-4bc1-b37d-b0186b867270.png)
![image](https://user-images.githubusercontent.com/55179520/201266762-785dbe78-dcf5-4872-b706-e2c28a6ac44e.png)

**Comment by hughbreslin** [comment, marked as answer]
Hi @chu551122 it looks like this is running as sh and not bash, have a look at this [issue](https://stackoverflow.com/questions/31075438/getting-error-bin-sh-1-syntax-error-unexpected) from googling. Please add `bash` before your make call and see if it works. If so we can try and find the root of the issue.
